domain/src/main/assets/test_prototype_exploration.json

```diff
@@ -0,0 +1,217 @@
+{
```

Comment:
I thought to make a single test file as it wouldn't make any sense having different files because for these will be in the same file so let's have only test exploration file. Sounds good?

Reply:
No, I think we should have separate files. We are likely going to use this same JSON file for all other drag-and-drop related tests, including the UI ones, so we wouldn't want to have other interactions there (otherwise we would need to navigate through unrelated interactions to get to the one we want to test). We have another exploration (prototype_exploration) for verifying all interactions in one go. In that way, we would have one exploration file that combines all interactions, and one exploration for each individual interaction to keep those tests targeted.